readthedocs/projects/models.py

```diff
@@ -235,6 +235,14 @@ class Project(models.Model):
             'This may slow down your page loads.',
         ),
     )
+    analytics_disabled = models.BooleanField(
+        _('Disable Analytics'),
+        default=False,
+        help_text=_(
+            'Disable Google Analytics completely for this project '
+            '(requires rebuilding documentation)',
+        ),
+    )
```

Comment:
Adding this new field will add some downtime when doing the deploy, this can be avoided by making it nullable, just remember to add something like this to the form https://github.com/readthedocs/readthedocs.org/pull/6996/files#diff-6115650dbb130936b82bd0aa90fd0030L39-L42 to avoid rendering it as a select box.

Reply:
This downtime can be avoided by deploying first to web-extra and running the migration. The servers running old code at that time should be fine.